descriptors/ModuleDescriptor-template.json

```diff
@@ -51,7 +51,8 @@
           "modulePermissions": [
             "change-manager.parsedrecords.get",
             "users.item.get",
-            "converter-storage.field-protection-settings.get",
+            "converter-storage.field-protection-settings.item.get",
+            "converter-storage.field-protection-settings.collection.get",
```

Comment:
We use only `field-protection-settings/marc` endpoint, so I believe that we should have only "converter-storage.field-protection-settings.collection.get" permission. Am I right?

Reply:
Indeed, in the `FieldProtectionSettingsClient` interface, there is only `field-protection-settings/marc`. Thanks for the comment. I fixed it.